modular_zubbers/code/modules/zantag_tickets/antag_ticket_multipliers.dm

```diff
@@ -0,0 +1,128 @@
+/datum/antagonist
+	//Loss multiplier for this antagonist type.
+	//Null means not actually an antagonist, and should give points.
+	//0 means don't give any points, but don't take any away.
+	var/antag_ticket_multiplier
```

Comment:
I feel like most of this could be simplified by setting it to 1 by default and writing any exceptions to this file, most of these antags have a multiplier of 1

Reply:
Update: I am stupid, this exists so things that aren't an antagonist but have antag datums aren't counted, like ashwalkers

Reply:
Some of these minor antagonist roles absolutely should not consume tickets. Space dragon carps, venus traps, and borers? Really? There's already no guarantee that ghosts will be available to join you as adds and now if you're a dragon or botanist traitor you have to worry if the ghosts that are present are willing to spend their antag tickets to join.